config/routes.rb

```diff
@@ -3,4 +3,12 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  
+   # Restful design routes
+  # resources :users, only: [:index, :show] do
+  #   resources :posts, only: [:index, :show]
+  root 'users#index'
+  get '/users/:id', to: 'users#show'
+  get 'users/:id/posts', to:'posts#index'
+  get 'users/:id/posts/:id', to:'posts#show'
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you the `resources` method for implementing your routes instead of the ` get` as this is so easy to implement routes you want in your app. You can implement your routes like this:
```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```